scripts/install_boost.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -ex
+
+BOOST_VERSION="$1"
+BOOST_MAJOR_MINOR=$(echo "${BOOST_VERSION}" | cut -d. -f-2)
+BOOST_MAJOR=$(echo "${BOOST_VERSION}" | cut -d. -f-1)
+BOOST_MINOR=$(echo "${BOOST_MAJOR_MINOR}" | cut -d. -f2-)
+BOOST_PATCH=$(echo "${BOOST_VERSION}" | cut -d. -f3-)
+BOOST_VERSION_U="${BOOST_MAJOR}_${BOOST_MINOR}_${BOOST_PATCH}"
+
+echo "using python : : /usr/bin/python${PYTHON_VERSION} ;\n" \
```

Comment:
```suggestion
echo "using python : : $(which python) ;\n" \
```
PYTHON_VERSION is undefined (crept in from my OCIO script). Could use this instead, or allow the boost python version to be explicitly defined.